src/components/Buttons/Button.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+function Button(props) {
+  const ButtonComponent = styled.button`
+    background: white;
+    color: red;
+
+    font-size: 1em;
+    margin: 1em;
+    padding: 0.25em 1em;
+    border: 2px solid red;
+    border-radius: 3px;
+  `;
+  return <ButtonComponent>{props.children}</ButtonComponent>;
+}
+
+function FilledButton(props) {
+  const ButtonComponent = styled.button`
+    background: red;
+    color: white;
+
+    font-size: 1em;
+    margin: 1em;
+    padding: 0.25em 1em;
+    border: 2px solid red;
+    border-radius: 3px;
+  `;
+  return <ButtonComponent>{props.children}</ButtonComponent>;
+}
+export { Button, FilledButton };
```

Comment:
You should avoid declaring a styled component inside your react components because you are dynamically creating a new component on every render. Please read this: https://styled-components.com/docs/faqs#why-should-i-avoid-declaring-styled-components-in-the-render-method